MFA Docs Changes (#6942)
auth0/docs: articles/multifactor-authentication/factors/otp.md

---
title: MFA - One Time Passwords
description: Using one time passwords with Auth0 MFA
toc: true
topics:
    - mfa
contentType:
  - index
useCase:
  - customize-mfa
---
# MFA with One Time Passwords (OTP)

> The principle behind OTP as a factor is fairly straightforward for the end user. They use an app, such as:

Reviewer: ```suggestion
To use One Time Passwords as an authenticator factor, users need an Authenticator app such as:
```

* Authy ([Google Play](https://play.google.com/store/apps/details?id=com.authy.authy) / [App Store](https://itunes.apple.com/us/app/authy/id494168017)).
* Google Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) / [App Store](https://itunes.apple.com/us/app/google-authenticator/id388497605)).
* Guardian ([Google Play](https://play.google.com/store/apps/details?id=com.auth0.guardian) / [App Store](https://itunes.apple.com/us/app/auth0-guardian/id1093447833)).
* Microsoft Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.azure.authenticator) / [App Store](https://itunes.apple.com/us/app/microsoft-authenticator/id983156458)).

![MFA Google Authenticator](/media/articles/multifactor-authentication/google-auth-scan-code.png)

> 

Reviewer: This image is the one we use for Google Authenticator, which we'll not be showing in the future (ony if you set google-authenticator), try enabling OTP and enrolling, you'll get a similar screen rendered by the Guardian widget.

> Upon signup, they can scan a code and set up the app, upon which it will begin generating one-time codes. Afterwards, the user can simply check the app for the current one-time code to enter when authenticating using this factor.

Reviewer 2: We don't mention recovery-code here, and we do for the others, but I guess it's fine!

> Your users will need to have in their posession a supported device for whichever OTP app you use.

Reviewer: ```suggestion
Your users will need to have an OTP Authenticator app installed in their mobile devices.
```

description: Using one time passwords with Auth0 MFA
toc: true
topics:
    - mfa
contentType:
  - index
useCase:
  - customize-mfa
---
# MFA with One Time Passwords (OTP)

To use one time passwords as an authentication factor, users need an Authenticator app such as:

* Authy ([Google Play](https://play.google.com/store/apps/details?id=com.authy.authy) / [App Store](https://itunes.apple.com/us/app/authy/id494168017)).
* Google Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) / [App Store](https://itunes.apple.com/us/app/google-authenticator/id388497605)).
* Guardian ([Google Play](https://play.google.com/store/apps/details?id=com.auth0.guardian) / [App Store](https://itunes.apple.com/us/app/auth0-guardian/id1093447833)).
* Microsoft Authenticator ([Google Play](https://play.google.com/store/apps/details?id=com.azure.authenticator) / [App Store](https://itunes.apple.com/us/app/microsoft-authenticator/id983156458)).

![MFA OTP Signup](/media/articles/multifactor-authentication/mfa-otp-setup.png)

Upon signup, they can scan a code and set up the app, upon which it will begin generating one-time codes. 

> Afterwards, when logging in to the app, the user can simply check the app for the current one-time code:

Reviewer: ```suggestion
Afterwards, when logging in to the app, the user can simply check the authenticator app for the current one-time code:
```
